src/Backends/Cubics/CubicBackend.h

```diff
@@ -200,11 +203,12 @@ class SRKBackend : public AbstractCubicBackend  {
                bool generate_SatL_and_SatV = true){
         std::vector<double> Tc, pc, acentric;
         N = fluid_identifiers.size();
+        components.resize(N);
         for (std::size_t i = 0; i < fluid_identifiers.size(); ++i){
-            CubicLibrary::CubicsValues val = CubicLibrary::get_cubic_values(fluid_identifiers[i]);
-            Tc.push_back(val.Tc);
-            pc.push_back(val.pc);
-            acentric.push_back(val.acentric);
+            components[i] = CubicLibrary::get_cubic_values(fluid_identifiers[i]);
```

Comment:
Looks good.  It would be perhaps a good idea to write a common setter function for Tc, pc, acentric for both of the cubic backends in the AbstractCubicBackend.  For now this is fine...